pkg/sfu/buffer/buffer.go

```diff
@@ -18,22 +18,20 @@ import (
 	"go.uber.org/atomic"
 )
 
-const (
-	ReportDelta = 1e9
-)
-
 type pendingPacket struct {
 	arrivalTime int64
 	packet      []byte
 }
 
 type ExtPacket struct {
-	Head      bool
-	Arrival   int64
-	Packet    *rtp.Packet
-	Payload   interface{}
-	KeyFrame  bool
-	RawPacket []byte
+	Head          bool
+	Arrival       int64
+	Packet        *rtp.Packet
+	Payload       interface{}
+	KeyFrame      bool
+	RawPacket     []byte
+	SpatialLayer  int32
```

Comment:
Recording spatial layer and temporal layer in `ExtPacket` which gets forwarded. `SpatialLayer` is forward looking for SVC style codecs. There will be a single buffer (i. e. single SSRC) which will have spatial layer information also.

NOTE: Thinking about including temporal layer in relay protocol, but not decided yet.